packages/nimble-components/src/wafer-map/tests/wafer-map-matrix.stories.ts

```diff
@@ -115,6 +117,16 @@ const componentWaferWithDieOrientation = ([
 >
 </nimble-wafer-map>`;
 
+const componentWaferResize = (
+    size: number
+): ViewTemplate => html`<nimble-wafer-map
+    style="width: ${size}px; height: ${size}px"
+    class="size${size}"

```

Comment:
doesn't look like the class is needed anymore

Reply:
applied